### ISSUE TITLE
attention_mask

### ISSUE THREAD

**Issue by wudixin96** [neosun100/DeepSeek-OCR-WebUI#27]
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:None for open-end generation.
